### ISSUE TITLE
Some MACROS missing from the header file

### ISSUE THREAD

**Issue by kaushikSR** [afosdick/ese-coursera-course1#3]
course.c file uses macros like BASE_16,BASE_10 but these are not defined in any header file.

**Comment by Arna-Maity** [comment]
@kaushikSR Yes exactly! These MACROS do come up as undefined during the build process.
